x-pack/plugins/fleet/public/components/enrollment_instructions/root_privileges_callout.test.tsx

```diff
@@ -7,15 +7,14 @@
 
 import React from 'react';
 
-import { cleanup, waitFor } from '@testing-library/react';
+import { waitFor } from '@testing-library/react';
 
 import { createFleetTestRendererMock } from '../../mock';
 
 import { RootPrivilegesCallout } from './root_privileges_callout';
 
 describe('RootPrivilegesCallout', () => {
   function render(rootIntegrations?: Array<{ name: string; title: string }>) {
-    cleanup();
```

Comment:
yields from from removing  this:
<img width="1281" alt="Screenshot 2024-11-22 at 18 00 07" src="https://github.com/user-attachments/assets/e4dc6ba5-20c4-4c6c-8230-890aa5a620c9">